### PR TITLE
update README and clean up obsolete file

### PR DESCRIPTION
**Pull request by dminca**
* always install Ansible via `pip` as that contains the latest Ansible
* remove obsolete empty file

Resolves:

Related:

Signed-off-by: Daniel Andrei Minca <mandrei17@gmail.com>

### CODE REVIEW

File: roles/hardware/tasks/main.yml [hunk 1]
```diff
@@ -15,6 +15,5 @@
     - "{{ ansible_product_name | replace(' ', '_') }}-{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
     - "{{ ansible_product_name | replace(' ', '_') }}-{{ ansible_distribution }}.yml"
     - "{{ ansible_product_name | replace(' ', '_') }}-{{ ansible_os_family }}.yml"
-    - default.yml
```
Comment (SISheogorath) [inline roles/hardware/tasks/main.yml]: By removing this, the playbook fails if no hardware specific changes are needed.

Comment (dminca) [reply roles/hardware/tasks/main.yml]: Oups! Reverting 😸

**Comment by SISheogorath** [comment]
Using ansible with DNF has multiple reasons.

Using `pip` conflicts with `dnf` installed. Also pip installed versions need to be updated in a manual way. the dnf setup will be updated with the automated update that is installed by this playbook

**Comment by dminca** [comment]
Thank you for clearing that up @SISheogorath 😃

**Comment by SISheogorath** [comment]
You're very welcome :)

**Pull request closed without merging**